uninstall.php

```diff
@@ -185,7 +185,7 @@ public function uninstall() {
 		delete_option( 'pll_language_from_content_available' );
 
 		// Delete transients.
-		delete_transient( 'pll_languages_list' );
+		delete_option( '_transient_pll_languages_list' ); // Force deletion of the transient from the options table in case external object cache is used.
```

Comment:
```suggestion
		delete_option( '_transient_pll_languages_list' ); // Force deletion of the transient from the options table even in case external object cache is used.
```